index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Portfolio</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="./style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    </head>
+    <body>
+        <section class="appbar">
+            <div class = "toolbar">
+                <div class="mylogo">
+                    My Logo
+                </div>
```

Comment:
- [ ] Kindly note that your logo should be inside an anchor tag, and once the user clicks on it, it redirects him to the homepage again, this will be easier for him when navigating through your app.
```suggestion
                <div class="mylogo">
                    <a href=" "> My Logo </a>
                </div>
```